src/index.html

```diff
@@ -3,12 +3,33 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img src="images/logo.png" alt="logo">
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="item1 hover is-active">apple</a></li>
+          <li class="nav__item"><a href="#" class="item2 hover">samsung</a></li>
+          <li class="nav__item"><a href="#" class="item3 hover">смартфоны и телефоны</a></li>
+          <li class="nav__item" >
+            <a href="#"
+             class="item4 hover"
+             data-qa="hover"
+            >ноутбуки и компьютеры</a>
```

Comment:
```suggestion
          <li class="nav__item" >
            <a 
              href="#"
              class="item4 hover"
              data-qa="hover"
            >
              ноутбуки и компьютеры
            </a>
```
Check code formatting
<img width="1178" alt="formatting" src="https://user-images.githubusercontent.com/94292654/172137056-bf32028c-934e-480e-bdd0-494c3f2a7dec.png">